src/store/modules/bitcoin.js

```diff
@@ -382,13 +387,33 @@ const actions = {
     }
   },
 
-  async getPrice({ commit }) {
+  async getPrice({ commit, state }) {
     const price = await API.get(
-      `${process.env.VUE_APP_MANAGER_API_URL}/v1/external/price`
+      `${process.env.VUE_APP_MANAGER_API_URL}/v1/external/price`,
+      {params: {currency: state.conversionCurrency}}
```

Comment:
To make full use of your precedent PRs (https://github.com/getumbrel/umbrel-manager/pull/56 and https://github.com/getumbrel/umbrel-manager/pull/57), I suggest to directly use the query here.
```suggestion
     `${process.env.VUE_APP_MANAGER_API_URL}/v1/external/price?currency=${state.conversionCurrency}`,
```